modules/ROOT/pages/index.adoc

```diff
@@ -81,8 +83,8 @@ The Object Store Connector operations that extend TTL are:
 |===
 
 
-* The minimum rolling TTL is 1 second.
-* The maximum rolling TTL is unlimited as long as the data continues to be accessed.
+* The minimum rolling TTL is 30 days (2592000 seconds).
```

Comment:
Minimum rolling TTL is 1 second.. This depends on the default TTL that is configured on the store. If a store has 1 second default TTL configured, then we extend the TTL on an item to currentTTL + 1 second.

Reply:
Not sure if we need to discuss that? 

We're going to nee to update this as well:
https://docs.mulesoft.com/object-store-connector/1.2/object-store-connector-reference#objectStorev2